src/config.rs

```diff
@@ -230,10 +308,8 @@ fn search_snare_conf() -> Option<PathBuf> {
     None
 }
 
-pub enum GenericOption<StorageT> {
-    Email(Lexeme<StorageT>),
-    MaxJobs(Lexeme<StorageT>),
-    Port(Lexeme<StorageT>),
-    ReposDir(Lexeme<StorageT>),
-    Secret(Lexeme<StorageT>),
+/// The configuration for a given repository.
+pub struct RConfig<'a> {
```

Comment:
How about `RepoConfig` for this type name?

Reply:
That's what I started off with, but the variable names all ended up being `rconf` so I shortened it. I suppose I could lengthen it again.

Reply:
Fixed in https://github.com/softdevteam/snare/pull/13/commits/32c4d2884fa532b6f5f0cca59d2eb21dfd10add8.

Reply:
Thanks. That's much clearer.

I'd have underscored the non-camel-case variable names, but won't insist.